lib/miq_pglogical/connection_handling.rb

```diff
@@ -2,10 +2,27 @@ class MiqPglogical
   module ConnectionHandling
     extend ActiveSupport::Concern
 
+    included do
+      delegate :logical_replication_supported?, :to => :class
+    end
+
     module ClassMethods
+      def logical_replication_supported?
+        return @logical_replication_supported if defined?(@logical_replication_supported)
+
+        is_superuser = ActiveRecord::Base.connection.select_value("select usesuper from pg_user where usename = CURRENT_USER;", "SQL")
+        warn "\e[33mWARNING: Current user is NOT a superuser, logical replication will not function.\e[0m" unless is_superuser
```

Comment:
I might leave out the colors here because this will likely go to the log.  Or perhaps if we do want colors we need a tty check.  Do we have access to a logger here?  If so, should we `logger.warn` this as well?

Reply:
done